src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -1461,7 +1461,11 @@ void InputProcessor::preScanReportingVariables()
         for (auto obj = epJSON_object.begin(); obj != epJSON_object.end(); ++obj) {
             json const &fields = obj.value();
             for (auto const &extensions : fields[extension_key]) {
-                addRecordToOutputVariableStructure("*", extensions.at("variable_or_meter_name"));
+                try {
```

Comment:
@mjwitte The input processor was blindly assuming that the report names would be present here.  We could add in logic to test for the key then try to add the record, but in this case a try block is essentially the same process, and quicker.  This change simply allows the code to continue, blank records have no effect at this point.